### PR TITLE
Fixed running of tests in parallel in some assemblies

### PR DESCRIPTION
**Pull request by fassadlr**
In some of the test assemblies the xunit.runner.json file was missing. This is required as we need to specify to xunit to run the tests serial. In other words, disable parallelization.

### CODE REVIEW

File: Stratis.Bitcoin.Features.BlockStore.Tests/LoopTests/BlockStoreLoopStepBaseTest.cs [hunk 1]
```diff
@@ -105,7 +104,7 @@ internal FluentBlockStoreLoop()
             this.ConfigureConnectionManager();
 
             this.BlockRepository = new BlockRepositoryInMemory();
-            this.dataFolder = TestBase.AssureEmptyDirAsDataFolder($"{AppContext.BaseDirectory}\\BlockStoreLoop");
+            this.dataFolder = TestBase.AssureEmptyDirAsDataFolder(Path.Combine(AppContext.BaseDirectory, "BlockStore"));
```
Comment (bokobza) [inline Stratis.Bitcoin.Features.BlockStore.Tests/LoopTests/BlockStoreLoopStepBaseTest.cs]: good stuff :-)

File: Stratis.Bitcoin.Features.BlockStore.Tests/LoopTests/BlockStoreLoopStepBaseTest.cs [hunk 2]
```diff
@@ -137,12 +136,8 @@ internal FluentBlockStoreLoop WithConcreteRepository(DataFolder dataFolder)
 
         private void ConfigureLogger()
         {
-            this.fullNodeLogger = new Mock<ILogger>();
```
Comment (bokobza) [inline Stratis.Bitcoin.Features.BlockStore.Tests/LoopTests/BlockStoreLoopStepBaseTest.cs]: why did you remove these? not needed?

Comment (fassadlr) [reply Stratis.Bitcoin.Features.BlockStore.Tests/LoopTests/BlockStoreLoopStepBaseTest.cs]: Yep, I wasn't using them at all :)

**Comment by bokobza** [comment]
awesome
